docs/src/main/sphinx/connector/memory.md

```diff
@@ -101,6 +101,3 @@ in the catalog file.
   or may return partial data.
 - When the coordinator fails/restarts, all metadata about tables is
   lost. The tables remain on the workers, but become inaccessible.
-- This connector does not work properly with multiple
```

Comment:
> Removed multiple coordinators mention in Memory connector docs

Could you avoid past tense in a commit title? https://cbea.ms/git-commit/ is our commit message guideline.

Reply:
@konovenski I mean a commit title, not PR title. We usually use `Rebase and merge` in this project.

Reply:
got it. done!